3RD_PARTY.md

```diff
@@ -5,6 +5,7 @@ A (very incomplete) list of libraries that make use of or expand on the capabili
 
 To add your project, open a pull request!
 
+- [Kafka Core](https://github.com/ffernandolima/confluent-kafka-core-dotnet) - Kafka Core empowers developers to build robust .NET applications on top of Confluent Kafka, focusing on simplicity, maintainability, and extensibility with intuitive abstractions and builders.
```

Comment:
We're maintaining this list in the order libraries were added. Can you please add it to the last?